docker/docs/builder/Dockerfile

```diff
@@ -6,22 +6,13 @@ ARG HTMLTEST_VERSION=0.17.0
 RUN CGO_ENABLED=0 go install github.com/wjdp/htmltest@v${HTMLTEST_VERSION} \
     && mv "${GOPATH}/bin/htmltest" /usr/bin/htmltest
 
-# we need clickhouse for running of the settings autogeneration script
-FROM clickhouse/clickhouse-server AS clickhouse-provider
+FROM node:20-bookworm-slim
 
-# nodejs 17 prefers ipv6 and is broken in our environment
-FROM node:20-alpine
+RUN apt-get update && \
+    apt install -y --no-install-recommends git openssh-server bash ca-certificates curl && \
+    rm -rf /var/lib/apt/lists/*:
 
-RUN apk add --no-cache git openssh bash curl libc6-compat
-
-# Copy ClickHouse binary from the clickhouse-provider stage
-COPY --from=clickhouse-provider /usr/bin/clickhouse /usr/bin/clickhouse
-ENV PATH=$PATH:/usr/bin
-
-# Despite using libc6-compat we are still missing some files
-# See https://github.com/ClickHouse/ClickHouse/issues/26350
-RUN git clone https://github.com/ClickHouse/libc-blobs.git -b master --depth=1
-RUN cp -r libc-blobs/aarch64/lib/* /lib/
+RUN curl https://clickhouse.com/ | sh
```

Comment:
Pro: it's a build-time action, so it's quick at the runtime.

Cons: the functions' documentation is potentially outdated.

Reply:
Hmmm, we probably don't want that. How often does it update?

Reply:
I thought that the binary for that is rebuild for every commit to master and pushed when ready, is it not?

Reply:
Not at all. It's rebuilt once in a while only when the `docker/docs/builder` directory is changed.

Reply:
Ah, sorry i was asking about the binary from `curl https://clickhouse.com/ | sh`. I see what you mean now. Well as long as curl is available and clickhouse will run in the container, which it should now that we are using `node:20-bookworm-slim` I can install the latest version from the settings generation script when docs check runs.

Reply:
Yes, I'll add it during the script for autogenerating settings. It gets run as part of `yarn build` in this image.